### ISSUE TITLE
Reference encoding

### ISSUE THREAD

**Issue by wwhurley** [balderdashy/sails-mongo#257]
When attempting to create new objects in Mongo and passing a reference to another object by ID the value that ends up in Mongo appears to be encoded in some strange way.

For instance, the object passed to .create() is:
```
{
  person: 5519c2785735b66d2ae9cc49
}
```

But when querying in the database I get:
```
{
  "person" : "T®íw¨!Ïp÷Ã\"j"
}
```

I've tried both .create() as well as .findOrCreate() as well as converting the string to an ObjectId using Model.mongo.objectId()

The model attributes are:
```
module.exports = {
  attributes : {
    date : {
      type : 'date',
      index : true
    },
    person : {
      model : 'People',
      index : true
    }
}
```

**Comment by listepo** [comment]
+1

**Comment by sailsbot** [comment]
Thanks for posting, @wwhurley.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
